pkg/util/etcdutil/migrate.go

```diff
@@ -208,6 +163,9 @@ func waitBootEtcdRemoved(etcdServiceIP string) error {
 	return err
 }
 
-func makeEtcdClusterURI(name string) string {
-	return fmt.Sprintf("/apis/%s/%s/namespaces/kube-system/%s/%s", spec.TPRGroup, spec.TPRVersion, spec.TPRKindPlural, name)
+func cleanupBootstrapEtcdService(restclient restclient.Interface) {
+	err := restclient.Delete().RequestURI("/api/v1/namespaces/kube-system/services/bootstrap-etcd-service").Do().Error()
```

Comment:
nit: It's a little fragile to rely on a fixed name here (bootstrap-etcd-service). It's possible the service manifest happened to have a diff name. If we load the actual object from the file, this would be easy enough to introspect prior to creating (and could also use regular client vs restclient). But this could also be a TODO